models/cow_protocol/tx_hash_labels/cow_protocol_tx_hash_labels_all.sql

```diff
@@ -0,0 +1,12 @@
+{{ config(
+    alias = 'all',
```

Comment:
Just a last thought, this table name will currently be set as `cow_protocol.all`, and I don't think that's what you had in mind? 
A good way to fix this would be to add schema definitions in the `dbt_project.yml` file

Reply:
> Just a last thought, this table name will currently be set as cow_protocol.all, and I don't think that's what you had in mind?

That's right, thanks! I've updated it.